generated/nirfmxinstr_restricted/nirfmxinstr_restricted.proto

```diff
@@ -0,0 +1,440 @@
+  rpc GetAttributeDesiredF64Array(GetAttributeDesiredF64ArrayRequest) returns (GetAttributeDesiredF64ArrayResponse);
+  rpc GetAttributeDesiredF32Array(GetAttributeDesiredF32ArrayRequest) returns (GetAttributeDesiredF32ArrayResponse);
+  rpc GetAttributeDesiredF32(GetAttributeDesiredF32Request) returns (GetAttributeDesiredF32Response);
+  rpc GetAttributeAuthor(GetAttributeAuthorRequest) returns (GetAttributeAuthorResponse);
+  rpc GetPrivilegeLevel(GetPrivilegeLevelRequest) returns (GetPrivilegeLevelResponse);
+  rpc RequestPrivilege(RequestPrivilegeRequest) returns (RequestPrivilegeResponse);
+  rpc SaveAllForRevert(SaveAllForRevertRequest) returns (SaveAllForRevertResponse);
+  rpc LoadAllForRevert(LoadAllForRevertRequest) returns (LoadAllForRevertResponse);
+  rpc GetInitiaitedSnapshotStrings(GetInitiaitedSnapshotStringsRequest) returns (GetInitiaitedSnapshotStringsResponse);
+  rpc GetSnapshotInfoFromCache(GetSnapshotInfoFromCacheRequest) returns (GetSnapshotInfoFromCacheResponse);
+  rpc GetLatestConfigurationSnapshot(GetLatestConfigurationSnapshotRequest) returns (GetLatestConfigurationSnapshotResponse);
+  rpc GetSnapshotState(GetSnapshotStateRequest) returns (GetSnapshotStateResponse);
+  rpc DeleteSnapshot(DeleteSnapshotRequest) returns (DeleteSnapshotResponse);
+  rpc ConvertForPowerUnitsUtility(ConvertForPowerUnitsUtilityRequest) returns (ConvertForPowerUnitsUtilityResponse);
+  rpc UnregisterSpecialClientSnapshotInterest(UnregisterSpecialClientSnapshotInterestRequest) returns (UnregisterSpecialClientSnapshotInterestResponse);
+  rpc GetCalibrationPlaneNames(GetCalibrationPlaneNamesRequest) returns (GetCalibrationPlaneNamesResponse);
+  rpc GetCalibrationPlaneEnabled(GetCalibrationPlaneEnabledRequest) returns (GetCalibrationPlaneEnabledResponse);
+  rpc GetExternalAttenuationTableNames(GetExternalAttenuationTableNamesRequest) returns (GetExternalAttenuationTableNamesResponse);
+  rpc GetActiveTableName(GetActiveTableNameRequest) returns (GetActiveTableNameResponse);
+  rpc GetSParameterExternalAttenuationType(GetSParameterExternalAttenuationTypeRequest) returns (GetSParameterExternalAttenuationTypeResponse);
+  rpc GetSignalConfigurationState64(GetSignalConfigurationState64Request) returns (GetSignalConfigurationState64Response);
+  rpc SetIOTraceStatus(SetIOTraceStatusRequest) returns (SetIOTraceStatusResponse);
+  rpc GetActiveResultName(GetActiveResultNameRequest) returns (GetActiveResultNameResponse);
+}
+
+message RegisterSpecialClientSnapshotInterestRequest {
+  string resource_name = 1;
+}
+
+message RegisterSpecialClientSnapshotInterestResponse {
+  int32 status = 1;
+}
+
+message GetOpenSessionsInformationRequest {
+  string resource_name = 1;
+}
+
+message GetOpenSessionsInformationResponse {
+  int32 status = 1;
+  string info_json = 2;
+}
+
+message GetRFmxVersionRequest {
+  nidevice_grpc.Session instrument = 1;
+}
+
+message GetRFmxVersionResponse {
+  int32 status = 1;
+  string r_fmx_version = 2;
```

Comment:
These should be `rfmx_version` of perhaps just `version`. I think the easiest way to fix this is using the `grpc_name` field in the `functions` metadata.

Reply:
Fixed in commit 5763b81.